### PR TITLE
Add Morocco capacities

### PR DESCRIPTION
**Pull request by veqtrus**
Source: http://www.one.org.ma/FR/pdf/Depliant%20Statistiques%20FR%202019.pdf

> Parc électrique national
> 
> | Centrale | Puissance (MW) |
> | --- | --- |
> | Usines hydrauliques* | 1306 |
> | STEP | 464 |
> | **Centrales thermiques vapeur** | **4716** |
> | Fioul | 600 |
> | Charbon (y compris JLEC) | 4116 |
> | Centrales turbines à gaz | 1110 |
> | Cycles combinés | 834 |
> | Thermique Diesel | 316 |
> | **Total thermique** | **6974** |
> | Eolien (ONEE et concessions) ** | 1220 |
> | solaire | 711 |
> | **Total** | **10677** |

The values seem to be consistent with those at https://data.med-tso.com/. A French speaker should double check that I got the meanings right.

I estimated the bounding box from Google Maps.

### CODE REVIEW

**Review by ajoga — APPROVED** [review]
Thank you for contributing! 

French speaker here, the numbers are good. Also the publication used is the last one available at this time (source= http://www.one.org.ma/FR/pages/interne.asp?esp=2&id1=10&id2=182&t2=1).

I need a core reviewer to confirm that bounding box with google maps are good enough, I personally used the [topogen script](https://github.com/tmrowco/electricitymap-contrib/tree/master/web) ; it calls the script that generates the bounding_box

**Comment by skovhus** [comment]
@FelixDQ can you let me know how you would validate the bounding boxes?

**Comment by FelixDQ** [comment]
I actually don't think we have documented this, but we don't want to add the bounding boxes manually. We can generate them using this script https://github.com/tmrowco/electricitymap-contrib/blob/master/web/generate-zone-bounding-boxes.js

It will look at the geometry and generate a bounding box from that.

File: config/zones.json [hunk 1]
```diff
@@ -3337,6 +3337,35 @@
     },
     "timezone": "Europe/Riga"
   },
+  "MA": {
+    "bounding_box": [
```
Comment (skovhus) [inline config/zones.json:3341]: @veqtrus can you update this to:
```
    "bounding_box": [
      [
        -17.51374332534519,
        20.919971097581723
      ],
      [
        -0.5319994709999492,
        36.426519149000015
      ]
    ],
```

You can create bounding boxes for new or existing zones in `config/zones.json` by running the following from the `web` folder:
1) Run: `docker-compose run --rm web ./topogen.sh`
2) Update the zone you want to update in `config/zones.json` with `"bounding_box": null`
3) Run: `node generate-zone-bounding-boxes.js`

Comment (veqtrus) [reply config/zones.json:3341]: Done

**Review by skovhus — APPROVED** [review]

**Pull request merged**